integration-tests/redis-devservices/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+quarkus.redis.devservices.port=6379
```

Comment:
How is this configuration file meant to be used with the one in `main`?

Reply:
Thanks for the explanation. I'd lean toward deleting this file and keep only one configuration file. I understand we want to override this for a specific test, in that case I'd use TestProfile for that. Are you okay with the suggestion?

Reply:
Ok for me, done.